src/test/java/org/example/SearchField.java

```diff
@@ -1,5 +1,81 @@
 package org.example;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import java.util.List;
+
+
 public class SearchField {
+    private WebDriver webDriver;
+
+    @BeforeClass
+    private void setDriver() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/webdriver/chromedriver.exe");
+        webDriver = new ChromeDriver();
+
+    }
+
+    @Test
+    private void searchFieldWritingTest() {
+        openAmazonWebPage();
+        String textTry= "kgvgvkyuoum,8lhlmu,ybjhuvbhjvlb.hylubyhv;bhjlv,khn;biuh";
```

Comment:
I would rename variable to randomTextForSearch or something a bit more suggestive